src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1546,53 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Address Mappings
+
+ - [List all address mappings](#List_all_address_mappinig)
+
+### List all address mappings
+
+```
+curl -XGET http://ip:port/mappings
+```
+
+Get all mappings from the [Recipients rewrite table](/server/config-recipientrewritetable.html)
+
+
+Response body:
+```
+{
+  "alias@domain.tld": [
+    {
+      "type": "Alias",
+      "mapping": "user@domain.tld"
+    },
+     {
+      "type": "Group",
+      "mapping": "group-user@domain.tld"
+    }
+  ],
+  "aliasdomain.tld": [
+    {
+      "type": "Domain",
+      "mapping": "realdomain.tld"
+    }
+  ],
+  "group@domain.tld": [
+    {
+      "type": "Address",
+      "mapping": "user@domain.tld"
+    }
+  ]
+}
+```
```

Comment:
please give me the various types that exist. For each type we should have a link to the routes managing them IMO